eq-author-api/src/validation/index.test.js

```diff
@@ -515,10 +588,11 @@ describe("schema validation", () => {
       const section = questionnaire.sections[0];
       section.title = "";
 
-      const validationErrors = validation(questionnaire);
-      const sectionErrors = validationErrors.sections[section.id].errors;
-      expect(sectionErrors).toHaveLength(1);
-      expect(sectionErrors[0]).toMatchObject({
+      const validationpageErrors = validation(questionnaire);
+      const sectionpageErrors =
+        validationpageErrors.sections[section.id].errors;
```

Comment:
validationpageErrors & sectionpageErrors look really odd.
The lack of proper camelcase disturbs me
How about? validationPageErrors & sectionPageErrors

Obviously not a show stopper

Reply:
Sorted